server/ansible/roles/pbench-server-deps/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Move to contrib - it should be part of a bootstrap role to set up an ansible environment, but it's not part of pbench-server setup.